src/renderRepoCard.js

```diff
@@ -75,7 +75,7 @@ const renderRepoCard = (repo, options = {}) => {
   `;
 
   const svgForks =
-    totalForks > 0 &&
+    forkCount > 0 &&
```

Comment:
FIXES BUG: previously we used totalForks which is kFormatter so it was failing for large repos